projects/core/components/primitive-textfield/textfield/textfield.component.ts

```diff
@@ -20,6 +20,7 @@ import {TUI_TEXTFIELD_HOST} from '@taiga-ui/core/tokens';
     providers: [TEXTFIELD_CONTROLLER_PROVIDER],
     host: {
         type: 'text',
+        '[attr.inputmode]': 'host.inputMode',
```

Comment:
We need to make sure users can override it. Look at how `id` is done.

Reply:
fixed